src/workers/types.ts

```diff
@@ -67,6 +67,7 @@ export type ChannelLoadEvent = {
   loadId: number;
   channelIndex: number[];
   data: Uint8Array[];
+  ranges: [number, number][];
```

Comment:
this, and the changes in IVolumeLoader, is how we pass the original intensity min/max back out from the volume loaders back to the volume channel storage itself.